OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -270,7 +281,7 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 
 		public string VoicePhraseForOrder(Actor self, Order order)
 		{
-			return (order.OrderString == "Harvest" || (order.OrderString == "Deliver" && !IsEmpty)) ? "Move" : null;
+			return (order.OrderString == "Harvest" || (order.OrderString == "Deliver" && !IsEmpty)) ? info.Voice : null;
```

Comment:
Does this mean you'd have the same (say, "Harvest") definition played when you order the harvester to harvest a resource and when you order this harvester to return with said resource? RA2 already had both Harvest and Return separated.